event-app-api/server.js

```diff
@@ -11,6 +11,7 @@ import {
   findGaps,
   isOverlapping,
 } from "./merge_intervals/merge_intervals.js";
+import { Op } from "sequelize";
 
```

Comment:
generally its best to have api response schema/template for all the end points. So that server always gurantees a standard response to client. example you can find smple one here https://kailashyogeshwar.medium.com/nodejs-elegant-api-response-for-your-microservices-bb1e778676a1